### ISSUE TITLE
Cannot duplicate section and subsection

### ISSUE THREAD

**Issue by hoangkhanh2002** [marinaglancy/moodle-format_flexsections#69]
Hi Marina - We can't duplicate sections and subsections, they get error. Hope you fix it soon. Thank you very much

**Comment by marinaglancy** [comment]
Thanks for reporting it. I did not realise that the "Duplicate section" functionality was added recently (in Moodle 4.2)

Currently this action does nothing at all in the format flexsections because of some weird parameters in the duplicate section URL, it was easy to fix, but when I fixed it I realised that the core implementation of the section duplicating also has problems. I created https://tracker.moodle.org/browse/MDL-80505 and also linked to existing bug report https://tracker.moodle.org/browse/MDL-78080 that describes more problems. 

I will look how I can address this problem in this plugin
